app/controllers/posts_controller.rb

```diff
@@ -30,4 +30,11 @@ def create
   def post_params
     params.require(:post).permit(:title, :text)
   end
+
+  def destroy
```

Comment:
- [ ] you should Use CanCanCan for Post authorization in Post Controller which will look something like:
```
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end

```